xarray/tests/test_cftime_offsets.py

```diff
@@ -380,6 +405,49 @@ def test_mul(offset, expected):
     assert offset * 3 == expected
 
 
+@pytest.mark.parametrize(
+    ("offset", "expected"),
+    [
+        (Day(), Hour(n=12)),
+        (Hour(), Minute(n=30)),
+        (Minute(), Second(n=30)),
+        (Second(), Millisecond(n=500)),
+        (Millisecond(), Microsecond(n=500)),
+    ],
+    ids=_id_func,
+)
+def test_mul_float(offset, expected):
+    assert offset * 0.5 == expected
+
+
+def test_mul_float_multiple_next_higher_resolution():
+    """Test more than one iteration through _next_higher_resolution is required."""
+    assert 1e-6 * Second() == Microsecond()
+    assert 1e-6 / 60 * Minute() == Microsecond()
```

Comment:
fails and I dont know why `assert 1e-6 /60/60 * Hour() == Microsecond()`

Reply:
Yeah, it fails in pandas as well:
```
>>> import pandas as pd
>>> hour = pd.offsets.Hour()
>>> 1e-6 / 60 / 60
2.7777777777777777e-10
>>> 1e-6 / 60 / 60 * hour
<0 * Hours>
```
I think it has to do with the default tolerances of `np.isclose`.  I wouldn't worry about it for now.

Reply:
ok